unit_tests/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION ${CMAKE_VERSION})
 project(unit_tests LANGUAGES C CXX)
 
-set(BUILD_UNIT_TESTS YES CACHE BOOL "Build unit tests")
+set(BUILD_UNIT_TESTS NO CACHE BOOL "Build unit tests")
```

Comment:
Would it make sense to keep unit tests building by default and exclude them in the Travis script? Or is there some reason other than Travis optimization that this decision was made?

Reply:
I feel it is better to not build these by default because of the following:
- I don't think the unit tests are run by many people if any
- Assuming the majority of people don't run them, the unit tests just increase compile time
- It's easy to enable them if a person wants them
- The unit test suite is not exhaustive enough to give confidence that things are working